docs/ref/forms/fields.txt

```diff
@@ -1222,6 +1226,13 @@ method::
             ...
             </select>
 
+    ``ModelChoiceField`` also has the attribute:
+
+    .. attribute:: iterator
+
+        By default, :class:`ModelChoiceIterator`. Change this attribute to
+        specify how choices are created from the queryset.
```

Comment:
With this in play, I'd drop the `Both fields have ...` paragraph above. The link down to contains the explanation.